src/login/logind-session.c

```diff
@@ -701,7 +701,7 @@ static int session_start_scope(Session *s, sd_bus_message *properties, sd_bus_er
 
 static int session_dispatch_stop_on_idle(sd_event_source *source, uint64_t t, void *userdata) {
         Session *s = userdata;
-        dual_timestamp ts;
+        dual_timestamp ts = DUAL_TIMESTAMP_NULL;
```

Comment:
Actually, the `session_get_idle_hint()` already does that, but I wanted to be defensive here and don't rely on this implementation detail. I mean I am just extra paranoid because uninitialised variables hurt us many times in the past.

Anyway, this can be safely dropped.